geomstats/learning/kmeans.py

```diff
@@ -8,10 +9,18 @@
 
 
 class RiemannianKMeans(TransformerMixin, ClusterMixin, BaseEstimator):
+    """Class for k-means clustering on manifolds.
+
+    Example
+    -------
+    Available example on the Poincaré Ball and Hypersphere manifolds
+    :mod:`examples.plot_kmeans_manifolds`
+
+    """
 
     def __init__(self, riemannian_metric, n_clusters=8, init='random',
                  tol=1e-2, mean_method='default', verbose=0):
-        """ K-Means algorithm using Riemannian manifolds
+        """k-means algorithm using Riemannian manifolds.
```

Comment:
Actually, there should be no docstring in the constructors:
see https://numpydoc.readthedocs.io/en/latest/format.html for example.

Could you fix this also, since we're on it?

Reply:
"""K-means